### ISSUE TITLE
totall loss

### ISSUE THREAD

**Issue by zeynali** [tensorflow/models#3338]
Hi ,
my dataset is roughly 50.000 images , and i fine-tune all layers of NasNet-a-large , but i don't know this is going correctly or no , The error is not less than 2.0 . is this normal?
